Shared/Samples/Add clustering feature reduction to a point feature layer/AddClusteringFeatureReductionToAPointFeatureLayerView.Model.swift

```diff
@@ -0,0 +1,177 @@
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A Zurich buildings web map.
+        let map = Map(
+            item: PortalItem(
+                portal: .arcGISOnline(connection: .anonymous),
+                id: .zurichBuildings
+            )
+        )
+        
+        /// A custom feature reduction for dynamically aggregating and
+        /// summarizing groups of features as the map scale changes.
+        private let clusteringFeatureReduction = makeCustomFeatureReduction(renderer: makeClassBreaksRenderer())
+        
+        /// The buildings feature layer in the web map.
+        var featureLayer: FeatureLayer? {
+            map.operationalLayers.first as? FeatureLayer
+        }
+        
+        /// A Boolean value indicating whether cluster labels are displayed.
+        var showsLabels: Bool {
+            didSet {
+                clusteringFeatureReduction.showsLabels = showsLabels
+            }
+        }
+        
+        /// The maximum scale of feature clusters.
+        /// - Note: The default value for max scale is 0.
+        var maxScale: Double {
+            didSet {
+                clusteringFeatureReduction.maxScale = maxScale
+            }
+        }
+        
+        /// The radius of feature clusters.
+        /// - Note: The default value for cluster radius is 60.
+        /// Larger radius allows more features to be grouped into a cluster.
+        var radius: Double {
+            didSet {
+                clusteringFeatureReduction.radius = radius
+            }
+        }
+        
+        init() {
+            // Set initial values for controls.
+            showsLabels = clusteringFeatureReduction.showsLabels
+            radius = clusteringFeatureReduction.radius
+            maxScale = clusteringFeatureReduction.maxScale ?? .zero
+        }
+        
+        /// Loads the web map and set up the feature layer.
+        func setup() async throws {
+            try await map.load()
+            featureLayer?.featureReduction = clusteringFeatureReduction
+        }
+        
+        /// Creates a class breaks renderer for the custom feature reduction
+        /// - Returns: A `ClassBreaksRenderer` object.
+        private static func makeClassBreaksRenderer() -> ClassBreaksRenderer {
+            // For each feature cluster with a given average building height,
+            // a color is assigned to each symbol.
+            let colors = [
+                (4, 251, 255),
+                (44, 211, 255),
+                (74, 181, 255),
+                (120, 135, 255),
+                (165, 90, 255),
+                (194, 61, 255),
+                (224, 31, 255),
+                (254, 1, 255)
+            ].map {
+                UIColor(red: $0.0 / 255, green: $0.1 / 255, blue: $0.2 / 255, alpha: 1)
+            }
+            
+            // Create a class break and a symbol to display the features
+            // in each value range.
```

Comment:
What page guide value is being used to wrap the text to next line? I see this in all ref doc in this pr. 
I generally use 100.

Reply:
We used to loosely follow a 80-column page guide, the same as the legal header's width.